code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -2407,6 +2407,28 @@
 	glass_name = "mug of apple cider juice"
 	glass_desc = "It's just spiced up apple juice. Sometimes the barista can't work miracles."
 
+/singleton/reagent/drink/notellamilkshake
+	name = "Notella Milkshake"
```

Comment:
not a fan of the name "notella", find an appropriate replacement

Reply:
Not sure what the issue is... "Notella" is already the name of the spread in game (the reagent is called choco-nut spread but the bottle is refferred to as a notella jar, there's already a food item in the game called notella bread slice, and i've also gotten several positive comments about it from people who saw it in the game. )